css/comp-page.css

```diff
@@ -0,0 +1,228 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;500;600;700&display=swap");
+:root {
+  --primary-colour: #11224d;
+  --secondary-colour: #5b84c4;
+  --primary-font-colour: #fff;
+  --secondary-font-colour: #000;
+  --font: "PT Sans", sans-serif;
+  --card-width: 15em;
+  --card-max-width: 90vw;
+  --card-img-height: 20em;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: "PT Sans", sans-serif;
+  font-weight: 400;
+}
+
+.site-nav {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  margin-left: 5em;
+  margin-top: 7em;
+  overflow-y: auto;
+}
+
+.nav-main-heading {
+  font-size: 1.3rem;
+  padding: 0.2em 0.5em;
+  border-radius: 0.2em;
+  display: block;
+  width: 100%;
+  color: #11224d;
```

Comment:
Use color variables instead of specifying direct color code.

Reply:
ack, done